akka-cluster/src/main/resources/reference.conf

```diff
@@ -495,6 +495,9 @@ akka.cluster.split-brain-resolver.lease-majority {
   # as an best effort to try to keep the majority side.
   acquire-lease-delay-for-minority = 2s
 
+  # Release the lease after this duration.
+  release-after = 40s
```

Comment:
Ok, so this _is_ default stable after * 2, but one thought: wasn't having it scaling with `stable-after` out of the box a good idea?

Reply:
Alright, lets go with a constant default like this 👍